dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -1101,5 +1107,9 @@ internal open class RumViewScope(
         private fun invertValue(value: Double): Double {
             return if (value == 0.0) 0.0 else 1.0 / value
         }
+
+        val navControllerActivityField = NavController::class.java.declaredFields.firstOrNull {
```

Comment:
Why did this get put at the bottom of the companion object instead of with the other `val`s?

Reply:
Mostly because I added it there, I could move it up but it doesn't really change anything IMO